src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Fuzzing strategy selection cron job. 
+
+	Runs multi-armed bandit experiments for fuzzing strategy selection.
+	In particular, this is a Boltzman Exploration (softmax) implementaion
+  of multi-armed bandit experiments. Queries from bigquery to update
+  multi-armed bandit probability values based on the new edges for various
+  combined strategies. In the upload_bandit_weights function, we can change
+  metric to be for edges, crash, features, or units. Currently based on new edges."""
+
+from datastore import data_types
+from datastore import ndb
+from google_cloud_utils import big_query
+from handlers import base_handler
+from libs import handler
+
+
+
+
+"""BigQuery query for calculating multi-armed bandit probabilities for
+various strategies using a Boltzman Exploration (softmax) model.
+
+See https://www.cs.mcgill.ca/~vkules/bandits.pdf
+"""
+MULTI_ARMED_BANDIT_PROB = """
```

Comment:
Nit prob/probs -> probability/probabilities throughout the file.

Reply:
Looks fixed in some other places but still an issue for this name.

On second thought, it may actually be best to give this a name that makes it more obvious that it's a query we intend to run, though. From the current name I'd almost expect this to be some fixed probability we use for calculations.